ViAn/GUI/mainwindow.cpp

```diff
@@ -608,3 +609,12 @@ void MainWindow::add_video_to_tree(MyQTreeWidgetItem *project, std::string fileP
     project->addChild(videoInTree);
     set_selected_video(videoInTree);
 }
+
```

Comment:
This has no comments.. please make sure to check these things before making a pull request